extensions/git/src/git.ts

```diff
@@ -1526,6 +1526,24 @@ export class Repository {
 		}
 	}
 
+	async dropStash(index?: number): Promise<void> {
```

Comment:
If there's no need, let's not make this argument optional.

Reply:
Done.